Code/Tools/ProjectManager/Source/ProjectUtils.h

```diff
@@ -17,7 +17,7 @@ namespace O3DE::ProjectManager
         bool RegisterProject(const QString& path);
         bool UnregisterProject(const QString& path);
         bool CopyProjectDialog(const QString& origPath, QWidget* parent = nullptr);
-        bool CopyProject(const QString& origPath, const QString& newPath);
+        bool CopyProject(const QString& origPath, const QString& newPath, QWidget* parent);
```

Comment:
nit: `QWidget* parent = nullptr`

Reply:
Did that on purpose to force the user to provide the argument, so that it won't become an independent window.